sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/ParDoLifecycleTest.java

```diff
@@ -205,7 +204,7 @@ public void after() {
   }
 
   @Test
-  @Category(NeedsRunner.class)
```

Comment:
Why not make SDK `ValidatesRunner` changes in a separate ticket/PR? it might affect other runners too.
Probably best to make this change first, and follow-up with the bug fix for the Spark runner.

Reply:
Yes good idea, to be sure I don't break the other runners.